content/en/security/identity_risks/_index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cloud Security Management Identity Risks
+kind: documentation
+further_reading:
+- link: "/security/cloud_security_management/"
+  tag: "Documentation"
+  text: "Learn more about Cloud Security Management"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">
+CSM Identity Risks is not available in this site.
+</div>
+{{< /site-region >}}
+
+<div class="alert alert-info">CSM Identity Risks is in beta.</div>
+
+Cloud Security Management Identity Risks (CSM Identity Risks) provides in-depth visibility into your organization's AWS IAM risks. Enabling you to detect and resolve identity risks on an ongoing basis, CSM Identity Risks helps secure your cloud infrastructure from IAM-based attacks.
+
+<div class="alert alert-info">At this time, CSM Identity Risks is available for AWS only.</div>
+
+## Setup
+
+To use CSM Identity Risks, you must [enable resource collection for AWS][2] and [enable CloudTrail logs forwarding][4]. If you've already done this, no additional setup is required.
+
+## Review and remediate identity risk detections
+
+Review your organization's active identity risk detections on the [Identity Risks Explorer page][3]. Use the **Group by** options to filter by **Identity Risks**, **Resources**, or **None** (individual detections). View additional details on the side panel by selecting a resource or detection.
```

Comment:
Review your organization's identity risks on the [Identity Risks][3] explorer. Use the **Group by** options to organize by **Identity Risks**, **Resources**, or **None** (individual detections). View additional details on the side panel by selecting an identity risk or a resource.